test/unit/backend_api_logic/routing_policy_test.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+module BackendApiLogic
+  class RoutingPolicyTest < ActiveSupport::TestCase
+    setup do
+      @service = FactoryBot.create(:service)
```

Comment:
```suggestion
      @service = FactoryBot.create(:simple_service)
```

The only difference between those 2 factories is that the account it creates is simple or with a bunch of stuff, and in this test you don't seem to want to use the account at all so it seems fair enough to make it as efficient as possible then :smile: